src/ontology/edits/oeo-physical.omn

```diff
@@ -477,6 +720,16 @@ Class: <http://openenergy-platform.org/ontology/oeo-physical/Wind>
         <http://purl.obolibrary.org/obo/BFO_0000015>
     
     
+Class: <http://openenergy-platform.org/ontology/oeo-physical/WindEnergyConverter>
```

Comment:
Rename to *WindEnergyConverterUnit* or *WindEnergyUnit* to prevent confusion with the energy converter device class?

Reply:
renamed to WindEnergyConvertingUnit